website/content/partials/api-docs/auth-method-params.mdx

```diff
@@ -98,10 +98,9 @@ for the auth method.
     alongside "kid" and "type". Setting the "kid" header here is not allowed;
     use `PrivateKey.KeyID`.
 
-  - `OIDCEnablePKCE` `(bool: true)` - When set to `true`, Nomad will include
+  - `OIDCEnablePKCE` `(bool: false)` - When set to `true`, Nomad will include
   [PKCE][] verification in the auth flow. Even with PKCE enabled in Nomad,
-  which is the default setting, you may still need to enable it in your OIDC
-  provider.
+  you may still need to enable it in your OIDC provider.
```

Comment:
@aimeeu this no longer being enabled by default (once again) affects your release notes in #25433 

I'm not touching `upgrade-specific.mdx` in this PR to avoid causing another merge conflict for you :upside_down_face: